.gitignore

```diff
@@ -82,4 +82,6 @@ dist/
 node_modules
 yarn-error.log
 
-.pre-release-scripts.txt
\ No newline at end of file
+dist/
+
+.pre-release-scripts.txt
```

Comment:
I don't think this should be ignored (what if we need to do pre-release stuff on kudet itself?)